### DISCUSSION TITLE
Is there a way to push Audit Logs (User Actions) to external service like Splunk or Kafka?

### DISCUSSION THREAD

**Discussion by m-harikiran** [apache/airflow#43811] [Q&A]
I'm looking for ways to push airflow audit logs to external service like Splunk or Kafka or even S3. This is to comply with our rules, that all audit logs should be available in one place for analysis.

Airflow provides REST API, but we don't want to poll it frequently and also we want this in near real-time as there will be notifications send to users and teams based on logs.

**Comment by raphaelauv** [comment]
what about CDC with debezium ?
